package.json

```diff
@@ -32,24 +32,33 @@
     "babel-loader": "^8.2.5",
     "copy-webpack-plugin": "^11.0.0",
     "core-js": "^3.25.0",
+    "css-loader": "^6.7.3",
     "eslint": "^8.23.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-config-standard-with-typescript": "^22.0.0",
     "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jsdoc": "^46.0.0",
     "eslint-plugin-n": "^15.2.5",
     "eslint-plugin-promise": "^6.0.1",
     "eslint-plugin-react": "^7.31.1",
+    "exports-loader": "^4.0.0",
     "prettier": "^2.7.1",
     "regenerator-runtime": "^0.13.9",
+    "style-loader": "^3.3.2",
     "typescript": "^4.8.2",
     "webpack": "^5.74.0",
     "webpack-cli": "^4.10.0",
     "webpack-dev-server": "^4.10.1"
   },
   "dependencies": {
+    "@azure/msal-browser": "^2.38.0",
+    "@azure/msal-react": "^1.5.9",
     "@emotion/react": "^11.10.4",
     "@emotion/styled": "^11.10.4",
+    "@mui/icons-material": "^5.14.0",
     "@mui/material": "^5.10.3",
+    "msal": "^1.4.18",
```

Comment:
`msal`and `msalx` aren't libraries that I used for Microsoft authentication... added on accident?

Reply:
Huh, okay I'll remove those